server/review-task/comment/comment.module.ts

```diff
@@ -13,7 +15,7 @@ export const CommentModel = MongooseModule.forFeature([
 ]);
 
 @Module({
-    imports: [CommentModel, UsersModule],
+    imports: [CommentModel, UsersModule, ConfigModule, AbilityModule],
```

Comment:
question: why the import of `ConfigModule` ?

Reply:
From what I understand when doing this task, the configModule is used to manage environment variables, in this case for users' tokens when logged in. I found some examples on aletheia of this security layer using the same, correct me if I'm wrong. As I said, I'm having difficulty testing using insomnia